openmetadata-ui/src/main/resources/ui/src/utils/AdvancedSearchUtils.tsx

```diff
@@ -137,9 +137,17 @@ export const renderAdvanceSearchButtons: RenderSettings['renderButton'] = (
   return <></>;
 };
 
+const getSearchLabel = (itemLabel: string, searchKey: string) => {
+  const regex = new RegExp(searchKey, 'gi');
+  const result = itemLabel.replace(regex, `<mark>${searchKey}</mark>`);
+
+  return result;
+};
+
 export const getSearchDropdownLabels = (
   optionsArray: string[],
-  checked: boolean
+  checked: boolean,
+  searchKey: string
```

Comment:
Make it option so that we don't need to pass it if not needed
```suggestion
  searchKey?: string
```